### PR TITLE
Add 2022 workshop vis entries to gallery

### PR DESCRIPTION
**Pull request by pazner**
Adds the 2022 vis contest entries to the gallery page.

### CODE REVIEW

File: src/gallery.md [hunk 1]
```diff
@@ -246,6 +265,24 @@ This page collects screenshots from various simulations based on MFEM. Additiona
 
 *Visualization of time-averaged mean flow from a compressible, DG Navier-Stokes solver using MFEM.  The geometry matches the internal flow domain of a cylindrical plasma torch where gas is injected into four small diamond ports. Image courtesy of Karl W. Schulz, as part of the 2021 MFEM Workshop Visualization Contest.*
 
+----
+
+[![](img/gallery/workshop22/gis_elevation_small.png)](img/gallery/workshop22/gis_elevation.png)
```
Comment (tzanio) [inline src/gallery.md:270]:
```suggestion
[![](img/gallery/workshop22/gis_elevation_small.png)](img/gallery/workshop22/gis_elevation.jpg)
```

Comment (pazner) [reply src/gallery.md]: Good catch!

**Comment by tzanio** [comment]
I suggest that we replace the two 2021 vis winners in the main page carousel with the new winners.

What do you think?

**Comment by tzanio** [comment]
Lets also remove `blastFEM` and `gis_elevation` to keep things more focused...

**Review by tzanio — APPROVED** [review]
Thanks @pazner !

**Review by hauten — APPROVED** [review]
no changes from me

**Pull request merged**